### ISSUE TITLE
Name picker not highlighting selected search name on Android 3.1 Honeycomb

### ISSUE THREAD

**Issue by ElimAdmin** [SparkDevNetwork/Rock#531]
In Chrome the name picker shows the drop down list of names, but selecting one results in an  exception.

HttpUnhandledException in System.Web
-FormatException in mscorlib.

**Comment by arranf** [comment]
@ElimAdmin Are you still having issues on Android v4.0+?

**Comment by nairdo** [comment]
I've spent some time looking into this and attempting to implement a "hover" feature (which is what the highlighting does on a system with a hoverable mouse) on a device that does not inherently support hover is not quite doable (without some hacking up the current typeahead.js implementation).  I've looked into [newer, forked versions of the typeahead.js library](https://github.com/corejavascript/typeahead.js) and tested some of their examples and it appears that they also do not support a hover feature on iOS devices (I'm testing using Safari on an iPad mini) except with their Scrollable Dropdown Menu.

If we ever decide to re-implement the typeahead feature we'll consider the implications of changing to use the scrollable dropdown menu, until then, we're going to label this an enhancement and close this issue.
